### PR TITLE
Docs- hyperthread note - JIRA gpdocs-41

### PR DESCRIPTION
**Pull request by lenapivotal**
Creates a note based on the research done on HT. 

JIRA [GPDOCS-41](https://jira.eng.vmware.com/browse/GPDOCS-41)

### CODE REVIEW

File: gpdb-doc/markdown/install_guide/platform-requirements.html.md [hunk 1]
```diff
@@ -151,6 +151,14 @@ The following table lists minimum recommended specifications for hardware server
 </table>
 </div>
 
+**Hyperthreading**
+
+Resource Groups - one of the key Greenplum Database features - can control transaction concurrency, CPU and memory resources, workload isolation, and dynamic bursting. 
+
+When using resource groups to control resource allocation in the server physical cores, it is beneficial to switch off hyperthreading (HT) in the server BIOS. Hyperthreading is `On` by default on Intel CPU cores.
```
Comment (kmacoskey) [inline gpdb-doc/markdown/install_guide/platform-requirements.html.md:158]:
```suggestion
When using resource groups to control resource allocation, it is beneficial to switch off hyperthreading (HT) in the server's BIOS. Hyperthreading is `On` by default on Intel CPU cores. 
```

It's not always control of physical cores.

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/install_guide/platform-requirements.html.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (xinzweb) [reply gpdb-doc/markdown/install_guide/platform-requirements.html.md]: Not sure we should stay with BIOS or UEFI. From https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.resmgmt.doc/GUID-4A7F14BF-8BE7-4DEF-A5FD-D7B4DC9336AE.html, we are still refer to the BIOS setting for the HT. Maybe we can keep consistent with that?

Comment (xinzweb) [reply gpdb-doc/markdown/install_guide/platform-requirements.html.md]: Shall we also mentioned by it's beneficial to turn off the HT? Something I am thinking would be:

```
When using resource groups to control resource allocation, for better resource isolation and protection between resource groups, it's beneficial to switch off Hyper-Threading (HT) in the server's BIOS if using Intel based systems. 
```

The reason for the above statement is that HT only applied to Intel, and it's their own trademark on their implementation of multi-threading on the physical cores. Please see https://www.intel.com/content/www/us/en/gaming/resources/hyper-threading.html for additional context.

**Review by xinzweb — CHANGES_REQUESTED** [review]
Please take a look at the additional comment.
The key info I'd like to call out, is the `trade-off` on tuning the HT configuration.

File: gpdb-doc/markdown/install_guide/platform-requirements.html.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dyozie) [reply gpdb-doc/markdown/install_guide/platform-requirements.html.md]: Is this hyperthreading note only intended for cloud deployments?  From the PR location I thought it applied to all (hardware + cloud) deployments.

**Pull request merged**